### ISSUE TITLE
As the Product Manager for MDN I want to see the performance is on par with the current implementation so we don’t take a hit on SEO

### ISSUE THREAD

**Issue by atopal** [mdn/sprints#1419]

**Comment by atopal** [comment]
# User story
As the Product Manager for MDN I want to see the performance is on par with the current implementation so we don’t take a hit on SEO

# Acceptance criteria
- [ ] Performance measurements have been set up for main KPI in Kuma Next
- [ ] Performance reported for main KPI in Speed-curve
- [ ] Performance is on par with with current Kuma

# Tasks
*If relevant, you can make a checklist for tasks.*

- [ ] This is an incomplete task.

**Comment by davidflanagan** [comment]
I'm pulling this user story into the new sprint, since we need to have this data before we go live.

The data we need *should* be being reported now, but I haven't seen it showing up in speedcurve yet, so this user story is to capture any remaining work toward getting that data.

**Comment by davidflanagan** [comment]
In issue #1450, @atopal wrote:

> Hey David, I looked into it. What we need is js-ie-load-event-end-measure and css-ie-load-event-end-measure. That's available in the non-react version, but missing in the react version

I think I enabled the measure that these two are based on in that other issue, but I guess I need to go one step further to get these two working. Hopefully this is just a Kuma issue, not another BoB or interactive examples issue.

**Comment by davidflanagan** [comment]
So unfortunately, this is both a Kuma and a BoB issue. I've filed two PRs:

https://github.com/mozilla/kuma/pull/5445
https://github.com/mdn/bob/pull/244

@schalkneethling: do you have time this sprint to push out another BoB release and regenerate the interactive examples?

**Comment by schalkneethling** [comment]
> do you have time this sprint to push out another BoB release and regenerate the interactive examples?

Done.

**Comment by davidflanagan** [comment]
Thanks, Schalk! 

I was able to get Speedcurve to run the react tests manually, and have verifiied that there is one datapoint shown on the speedcurve charts. I guess we won't get a line or a curve until we've got another data point.  I also noticed that we were running the non-react checks 3 times a day, but the react ones only once a day. I've bumped that back up to 3x/day so we get some data more quickly.

It will be interesting to see what LUX adds to this...

I've check off the first two tasks for this user story. If we can verify that performance is on-par, then we (probably Kadir, actually) can close this.

**Comment by atopal** [comment]
Can confirm that it's showing up on Speedcurve. I'd give it 2 days before we call performance parity, but the single data point does look good so far (considering the 1+ second variation on Speedcurve for synthetic tests)

**Comment by atopal** [comment]
hmm, the data for JS pages seems to be fine, but the CSS page has only reported a single data point for the react version 2 days ago: it's https://beta.developer.mozilla.org/en-US/docs/Web/CSS/box-shadow

**Comment by davidflanagan** [comment]
Even the non-react reporting is flaky for that test, with a bunch of dropped data points.

The one data point we do have for the beta site on that test is from the time I manually ran the tests. I'm trying that again now, to see if we get another data point from it.

Given that the page is reporting data, however, I don't know how I could debug this. It seems likely to be a load-related issue or a bug on speedcurve's side?

I'm hoping that once we ask people to start trying out the beta site, that LUX data will save us...

**Comment by davidflanagan** [comment]
The data is still fragmentary, but at least we've got some now. It is hard to tell with speedcurve, but it looks to me as if there is not a performance regression. What do you think @atopal?

<img width="711" alt="Screen Shot 2019-06-07 at 5 05 49 PM" src="https://user-images.githubusercontent.com/516914/59139250-c3f77000-8946-11e9-9691-436bdb81ecb1.png">

<img width="667" alt="Screen Shot 2019-06-07 at 5 06 12 PM" src="https://user-images.githubusercontent.com/516914/59139253-c6f26080-8946-11e9-845c-f6063be82c5e.png">

**Comment by davidflanagan** [comment]
We discussed this at the sprint meeting this morning and Kadir agreed that the data show no significant performance regression. So I'm marking this as done and closing it. See issue #1703 for a LUX-related followup issue.
